src/foam/nanos/crunch/CrunchService.js

```diff
@@ -32,6 +32,52 @@ foam.INTERFACE({
         }
       ]
     },
+    {
+      name: 'getCapabilityPath',
+      documentation: `
+        getGrantPath provides an array of capability objects representing
+        the list of capabilities required to grant the desired capability.
+      `,
+      async: true,
+      type: 'List',
+      args: [
+        {
+          name: 'x',
+          type: 'Context'
+        },
+        {
+          name: 'sourceId',
+          type: 'String'
+        },
+        {
+          name: 'filterGrantedUCJ',
+          type: 'boolean'
+        }
+      ]
+    },
+    {
+      name: 'getMultipleCapabilityPath',
+      documentation: `
+        getGrantPath provides an array of capability objects representing
+        the list of capabilities required to grant the desired capability.
+      `,
+      async: true,
+      type: 'List',
+      args: [
+        {
+          name: 'x',
+          type: 'Context'
+        },
+        {
+          name: 'capabilities',
```

Comment:
is this a list of capabilityId ? 
If so then perhaps the above could be 
capabilityId and this capabilityIds 
or is a capabilityId a capability and I'm using the wrong term.  Then the above could be capability, and this one capabilities.   Just trying to be consistent.